test/fw/ptl/utils/plugins/ptl_test_db.py

```diff
@@ -1725,8 +1726,10 @@ def __create_data(self, test, err=None, status=None):
         testdata['module'] = _test.__module__
         testdata['testcase'] = getattr(_test, '_testMethodName', '<unknown>')
         testdata['testdoc'] = getattr(_test, '_testMethodDoc', '<unknown>')
-        testdata['start_time'] = getattr(test, 'start_time', 0)
-        testdata['end_time'] = getattr(test, 'end_time', 0)
+        testdata['start_time'] = getattr(test, 'start_time',
```

Comment:
call datetime.datetime.now() just once, store its value and use it for both start_time and end_time